src/app/players/sidebar/sidebar.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { SidebarComponent } from './sidebar.component';
+
+describe('SidebarComponent', () => {
```

Comment:
No test cases are added

Reply:
I Will send a separate PR for test cases.